### PR TITLE
Charmed k8s : Disable the log collection in EndPxBackupTorpedoTest untill we find a way to ssh from juju

### PR DESCRIPTION
**Pull request by sn-px**
**What this PR does / why we need it**:
Disable the log collection in EndPxBackupTorpedoTest untill we find a way to ssh from juju

**Which issue(s) this PR fixes** (optional)
Closes #https://purestorage.atlassian.net/browse/PB-6282

**Special notes for your reviewer**:
https://jenkins.pwx.dev.purestorage.com/job/Users/job/santoshn/job/byoc-charm/17/consoleFull
https://jenkins.pwx.dev.purestorage.com/job/portworx-backup/job/system-tests/job/basic-system-test/job/px/job/s3/job/px-backup-system-test-vanilla/696/execution/node/365/log/?consoleFull

https://aetos.pwx.purestorage.com/resultSet/testSetID/598206

### CODE REVIEW

**Review by kphalgun-px — APPROVED** [review]
LGTM

File: tests/common.go [hunk 1]
```diff
@@ -7719,7 +7719,8 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 		log.FailOnError(err, "failed to switch context to source cluster")
 	}()
 	masterNodes := node.GetMasterNodes()
```
Comment (ak-px) [inline tests/common.go:7721]: @sn-px , Can you remove the above defer block at line 7717 in this PR.
Currently its trying to fetch the px-backup logs from destination cluster.

please find the below error:
https://aetos.pwx.purestorage.com/resultSet/testSetID/590948/testCaseID/3624513

**Review by ak-px — CHANGES_REQUESTED** [review]

File: tests/common.go (hunk 1 above)
Comment (sn-px) [reply tests/common.go:7717]: done

File: tests/common.go [hunk 2]
```diff
@@ -7714,12 +7714,12 @@ func EndPxBackupTorpedoTest(contexts []*scheduler.Context) {
 		log.Errorf("Error in deleting namespaces created by the testcase. Err: %v", err.Error())
 	}
 
-	defer func() {
-		err := SetSourceKubeConfig()
-		log.FailOnError(err, "failed to switch context to source cluster")
-	}()
```
Comment (mkoppal-px) [inline tests/common.go:7720]: Move it above and do not delete it

**Review by ak-px — APPROVED** [review]
LGTM

**Pull request merged**